### PR TITLE
Changing event duration in dashboard view

### LINKED ISSUE #1883: Fix dashboard view to be more appropriate
* Fix date to show starting and end data. Maked date in one row and the time directly below the date. Reduce width of the column.
* Expand the width of the column "Tickets" and reduce the column width of other column respectively. The goal is that each ticket name and the ticket numbers fit in one row also when ticket names are long.
* Ensure the view works on mobile views

Compare: https://eventyay.com/events/live

screenshot from 2019-01-18 00-32-53

### PR DESCRIPTION
**Pull request by uds5501**
- Created a local component for dashboard named cell-local-date
- Changed event controller so that new component routes to dashboard

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
This resolves the issue of visibility of only starting of an event. After this PR, the duration format is improved

#### Changes proposed in this pull request:

- Creation of a new local component ``local-cell-date``
- Modification of ``event`` controller to route to this new component
- Using ``records.startsAt`` as a conditional

**UI Change**
image

Fixes #1883

### CODE REVIEW

**Comment by uds5501** [comment]
@mariobehling @CosmicCoder96 @ritikamotwani Please review

**Comment by mariobehling** [comment]
Please make the "to" non capital. 

Travis is not building. Please check and resolve. Thanks!

File: app/controllers/events/list.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ export default Controller.extend({
     },
     {
       propertyName : 'startsAt',
-      template     : 'components/ui-table/cell/cell-simple-date',
+      template     : 'components/ui-table/cell/cell-local-date',
```
Comment (ritikamotwani) [inline app/controllers/events/list.js]: Change the name of the file to cell-event-date

File: app/templates/components/ui-table/cell/cell-local-date.hbs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{{#if record.startsAt}}
+  <div>
+    {{moment-format record.startsAt 'MMMM DD, YYYY - hh:mm A'}}
```
Comment (ritikamotwani) [inline app/templates/components/ui-table/cell/cell-local-date.hbs]: This is not the correct way. If you're adding the format here and calling the model variable again here what is the point of writing the propertyName and dateFormat in the controller?
Use something like (get record column.propertyName) in the template.
Use the file app/templates/components/ui-table/cell/cell-simple-date.hbs as reference.

File: tests/integration/components/ui-table/cell/cell-local-date-test.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    // Template block usage:
+    await render(hbs`
+      {{#ui-table/cell/cell-local-date}}
+        template block text
+      {{/ui-table/cell/cell-local-date}}
+    `);
+
+    assert.equal(this.element.textContent.trim(), 'template block text');
+  });
+});
```
Comment (ritikamotwani) [inline tests/integration/components/ui-table/cell/cell-local-date-test.js]: Change the component test file. This is just the default file you get when you generate a component.

Comment (uds5501) [reply tests/integration/components/ui-table/cell/cell-local-date-test.js]: Okay. Doing so

**Comment by uds5501** [comment]
@ritikamotwani @mariobehling @CosmicCoder96 kindly review now

**Comment by uds5501** [comment]
@mariobehling @ritikamotwani please review

File: app/templates/components/ui-table/cell/cell-event-date.hbs [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{{#if record.startsAt}}
```
Comment (CosmicCoder96) [inline app/templates/components/ui-table/cell/cell-event-date.hbs]: also check for `records.endsAt`

**Comment by uds5501** [comment]
@CosmicCoder96 I have addressed your request and committed accordingly. Kindly review!

File: app/templates/components/ui-table/cell/cell-event-date.hbs [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  {{#if record.endsAt}}
+    <div>
+      {{moment-format record.startsAt 'MMMM DD, YYYY - hh:mm A'}}
+    </div>
+    to
```
[the remark this answers is not in the document]
Comment (uds5501) [reply app/templates/components/ui-table/cell/cell-event-date.hbs]: @CosmicCoder96 I actually changed the `To` to `to` after the request of @mariobehling , so does it need to be discussed first?
Also, I will squash my commits right away

[the remark this answers is not in the document]
Comment (CosmicCoder96) [reply app/templates/components/ui-table/cell/cell-event-date.hbs]: @uds5501 ignore this.

**Comment by CosmicCoder96** [comment]
Commits were not squashed, ensure this doesn't happen again.

**Pull request merged**